addons/account_payment/views/account_portal_templates.xml

```diff
@@ -1,12 +1,70 @@
 <odoo>
+    <template id="portal_my_home_overdue_invoice" name="Portal layout : overdue invoices" inherit_id="portal.portal_breadcrumbs" priority="30">
```

Comment:
Extra space before the ":" ?

Reply:
It's just that most developers are french speaking and make the common mistake of adding the extra space in English.